common/config/config-distribution.rb

```diff
@@ -63,6 +63,9 @@ def self.get_preferred_config_path
     if java.lang.System.getProperty("aspace.config")
       # Explicit Java property
       java.lang.System.getProperty("aspace.config")
+    elsif ENV['ASPACE_CONFIG'] && File.exist?(ENV['ASPACE_CONFIG'])
```

Comment:
Ah, sorry, the commit message is confusing... "config.rb" should just be "the config file". 
They want to be able to put the config file outside of aspace, so something like :  "/etc/archivesspace_config.rb". 

So, the variable is the full filepath ( with file name ), not just the path to a "config.rb" file....